### DISCUSSION TITLE
Automatically remove new device check after a period of time

### DISCUSSION THREAD

**Discussion by wild9** [netalertx/NetAlertX#564] [Q&A]
Hi, apologies if I have missed something obvious but I have been looking for some time and not found a solution:

When a device is discovered it is marked as a new device, is there any sort of mechanism to remove this flag after a certain period of time? Or even a way to bulk remove it from devices rather than click through their properties one by one? I see there is a maintenance option to delete the entire device after a period of time but not to keep the device but just mark it as no longer new.

**Comment by jokob-sk** [comment]
Hi @wild9 ,

You could disable this flag completely when a new device is discovered:

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/93efda2f-01a2-404d-b441-11f70701edaf)

New devices will still be discovered and reported by the app, just the flag `New Device` will be set to `false`.

Also, I'm collecting feedback on an upcoming feature I'm considering expanding on, Workflows, which might cover your use case in the future: #562 Please add your comments in that thread.

You can also bulk-edit devices via the CSV export functionality: https://github.com/jokob-sk/Pi.Alert/blob/main/docs/DEVICES_BULK_EDITING.md

**Reply by wild9** [reply]
Thanks, I think in that case I possibly just don't fully understand the purpose of the check box.  Will I still be alerted (assuming my notifications are configured correctly) when new devices are discovered on the network even if I uncheck the option you mentioned?

**Reply by jokob-sk** [reply]
Yes, you will. The only purpose of this checkbox is to show the `New` status and to allow filtering of devices based on this status.

I've added this tooltip to clarify the usage:

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/74b2506b-8e9c-4753-8eed-33fe6cf1c4b5)

**Comment by stanthewizzard** [comment]
Can the new device be automatically unchecked after a period ?
And or
When multi edit is there a « all » than can be choose ?
Thanks

**Reply by jokob-sk** [reply]
**Select all** / **Remove all** buttons: This should be available in the next release. If you can, please have a look at the `netalertx-dev` docker image, in about 15 minutes (or after the last [action finishes](https://github.com/jokob-sk/NetAlertX/actions)) from now. 

It would be great if you could test this (backup everything first or use a new container) on your end by switching to the above image and letting me know if the issue was resolved.

Thanks in advance,
j

![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/fd2f7383-996e-4905-be3f-e0f7e212f661)
